.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,11 +1,40 @@
-<!-- INSTRUCTION: Your Pull Request name should start with one of the following tags -->
+<!-- This is a pull request template, you do not need to uncomment or remove the comments, they won't show up in the PR text. -->
+<!-- Your Pull Request name should start with one of the following tags -->
 <!-- [NEW] For new features -->
 <!-- [FIX] For bug fixes -->
 <!-- [BREAK] For pull requests including breaking changes -->
 
-<!-- INSTRUCTION: Inform the issue number that this PR closes, or remove the line below -->
-Closes #ISSUE_NUMBER
+## Proposed changes
+<!-- Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue below. -->
 
-<!-- INSTRUCTION: Link to a https://github.com/RocketChat/docs PR with added/updated documentation or an update to the missing/outdated documentation list, see https://rocket.chat/docs/contributing/documentation/  -->
+## Closes issue(s)
+<!-- Link the issues being closed by this PR. For example, you can use #594 if this PR closes issue number 594 -->
```

Comment:
Will this close the issues without the `Closes` prefix?

Reply:
This is just meant to tag the issues that could be related to the PR. I will remove the `Closes` prefix